### PR TITLE
Penv 103 common scripts submodule

### PR DESCRIPTION
**Pull request by megge-dream**
**- What I did**
Move common for every app scripts to insolar-scripts folder (it will became separate repo later)

**- How I did it**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: insolar-scripts/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# how to start launchnet and monitor
+
+To run launchnet, you must provide several files:
+
+`/scripts/insolard/bootstrap_template.yaml` - template for bootstrap config
```
Comment (Tsovak) [inline insolar-scripts/README.md]: `/scripts` or ` ./insolar-scripts/` ?
the path starts with `/`. do you mean absolute path?

File: scripts/insolard/generate_initial_data.sh [hunk 2]
```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+set -e
+
+# configurable vars
+INSOLAR_ARTIFACTS_DIR=${INSOLAR_ARTIFACTS_DIR:-".artifacts"}/
+LAUNCHNET_BASE_DIR=${LAUNCHNET_BASE_DIR:-"${INSOLAR_ARTIFACTS_DIR}launchnet"}/
```
Comment (Tsovak) [inline scripts/insolard/generate_initial_data.sh]: set slash between  INSOLAR_ARTIFACTS_DIR and launchnet as `${INSOLAR_ARTIFACTS_DIR}/launchnet"}/`

POSIX systems able to work correctly even if you pass double slash in the path. for example `ls -al /tmp//folder/`

File: scripts/insolard/generate_initial_data.sh [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+for (( b = 0; b < 1; b++ ))
+do
+bin/insolar gen-key-pair --target=user > ${CONFIGS_DIR}funds_${b}_member_keys.json
+done
+
+for (( b = 0; b < 8; b++ ))
+do
+bin/insolar gen-key-pair --target=user > ${CONFIGS_DIR}enterprise_${b}_member_keys.json
+done
+
```
Comment (Tsovak) [inline scripts/insolard/generate_initial_data.sh]: set slash between var and paths

Comment (megge-dream) [reply scripts/insolard/generate_initial_data.sh]: done

File: scripts/insolard/generate_initial_data.sh (hunk 2 above)
Comment (megge-dream) [reply scripts/insolard/generate_initial_data.sh]: done

File: insolar-scripts/README.md (hunk 1 above)
Comment (megge-dream) [reply insolar-scripts/README.md]: no, its relative path, fix it

**Pull request merged**